### ISSUE TITLE
i18next-parser applying wrong translation value from original locale files

### ISSUE THREAD

**Issue by Mankee** [i18next/i18next-parser#43]
Suppose I have a key 'foo_bar' in my application and it is currently translated for locale EN  but not for FR, ES (which are also supported locales in this app). The parser will grab the value from the EN locale and apply that value to FR, ES parsed locales.

I am trying to use the i18next-parser as a coverage report tool to see what key-value pairs still need translations for other locales other than our default english one. I was expecting keys that do not have have a translation in a particular locale to be parsed by the tool with an empty string as it's default value.

```
locales
│
└─── en
│  └─── translation.json
│
└─── fr
│  └─── translation.json
│
└─── es
     └─── translation.json
```

**original locales file**

```
locales/en/translation.json = {"foo_bar": "test translation", "welcome": "hello"}
locales/fr/translation.json = {"welcome": "bonjour"}
locales/es/translation.json = {"welcome": "hola"}
```
# Expected

**parsed locales file**

```
locales/en/translation.json = {"foo_bar"": "test translation", "welcome": "hello"}
locales/fr/translation.json = {"foo_bar": "", "welcome": "bonjour"}
locales/de/translation.json = {"foo_bar": "", "welcome": "hola"}
```
# Actual

**parsed locales file**

```
locales/en/translation.json = {"foo_bar": "test translation", "welcome": "hello"}
locales/fr/translation.json = {"foo_bar": "test translation",  "welcome": "bonjour"}
locales/de/translation.json = {"foo_bar": "test translation", "welcome": "hola"}
```

**Comment by karellm** [comment]
@Mankee Thanks for the detailed issue. Could you make a PR? I don't have the time currently to implement this but will review PRs.

**Comment by Mankee** [comment]
@karellm I have not digged too much into the source but if I have time I will try and submit a PR for this. Just wondering if you were able to reproduce my findings? I think the workaround for this in the meantime is to just run the i18next-parser per locale instead of specifying an array of locales in the config.

**Comment by Mankee** [comment]
@karellm I think I found the issue. The for loop that generates the translationHash object needs to run for each locale, otherwise you get translation values from whatever locale had a matching key value pair for that translation. I hope this makes sense. I am going to submit a PR that basically just moves the translationHash for loop inside the for loop that iterates over each locale. Feel free to reject if you have a better idea how to accomplish this.

**Comment by karellm** [comment]
This is addressed in #72, I wrote a passing test case. Closing this for now.
